### PR TITLE
[COM-394] SWC for all

### PR DESCRIPTION
**Pull request by tkanzakic**
**Checklist**
- [X] New feature (non-breaking change which adds functionality)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)
https://impulsumvc.atlassian.net/browse/COM-394

### CODE REVIEW

File: packages/cloud-kms/jest.config.js [hunk 1]
```diff
@@ -82,9 +82,9 @@ module.exports = {
   // ],
 
   // A map from regular expressions to module names that allow to stub out resources with a single module
-  moduleNameMapper: {
-    "^.+\\.module\\.css$": "identity-obj-proxy",
-  },
+  // moduleNameMapper: {
+  //   '^.+\\.module\\.css$': 'identity-obj-proxy',
+  // },
```
Comment (guilean) [inline packages/cloud-kms/jest.config.js:87]: Do we want this commented?

File: packages/analytics/jest.config.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ module.exports = {
 
   // A map from regular expressions to module names that allow to stub out resources with a single module
   moduleNameMapper: {
-    "^.+\\.module\\.css$": "identity-obj-proxy",
+    '^.+\\.module\\.css$': 'identity-obj-proxy',
   },
```
Comment (guilean) [inline packages/analytics/jest.config.js:87]: Do we want this uncommented?

**Review by guilean — APPROVED** [review]

File: packages/cloud-kms/jest.config.js (hunk 1 above)
Comment (tkanzakic) [reply packages/cloud-kms/jest.config.js:87]: yes

File: packages/analytics/jest.config.js (hunk 2 above)
Comment (tkanzakic) [reply packages/analytics/jest.config.js:87]: no

**Pull request merged**